code/game/objects/effects/temporary_visuals/miscellaneous.dm

```diff
@@ -159,6 +159,13 @@
 	layer = ABOVE_LYING_MOB_LAYER
 	duration = WRAITH_BANISH_BASE_DURATION+1 //So we don't delete our contents early
 
+/obj/effect/temp_visual/order/temporary_supply_beacon
+	name = "temp supply beacon"
+	icon = 'icons/misc/mark.dmi'
+	icon_state_on = "x3"
+	layer = UNDERFLOOR_OBJ_LAYER
```

Comment:
The layer doesn't work when it's added to an hud it seems, so this is pretty ugly

Reply:
ask slotbot for help he messed with layers

Reply:
use planes